lib/mox.ex

```diff
@@ -670,22 +679,33 @@ defmodule Mox do
 
       allow(MyMock, self(), SomeChildProcess)
 
+  If the process is not yet started at the moment of allowance definition,
+  it might be allowed as a lazy call, assuming at the moment of validation
+  it would have been started. If the function cannot be resolved to a `pid`
+  during call dispatch, the expectation would not succeed.
```

Comment:
```suggestion
  If the process is not yet started at the moment of allowance definition,
  it might be allowed as a function, assuming at the moment of invocation
  it would have been started. If the function cannot be resolved to a `pid`
  during invocation, the expectation will not succeed.
```